README.md

```diff
@@ -50,6 +54,29 @@ sls deploy
 ```
 5. 👀 Head to the CloudWatch section of the AWS Console to check out your new dashboards 📊 and alarms ⏰ !
 
+
+## Getting Started - CloudFormation Macro
+
+1. Deploying to your account (via the console)
+Go to the [AWS Application Repository](https://serverlessrepo.aws.amazon.com/applications) and click the Deploy button.
```

Comment:
We will have to add the correct link. It might be a good idea to publish the macro as a pre-release from this branch to validate the whole setup and know this URL.